client/src/components/Autocomplete.js

```diff
@@ -184,7 +187,7 @@ export default class Autocomplete extends Component {
 					+ 'list { ' + this.props.fields + '}'
 					+ '}'
 			let variableDef = '($query: ' + resourceName + 'SearchQuery)'
-			let queryVars = {text: value.value + "*", pageSize: 25}
+			let queryVars = {text: value.value + "*", pageSize: 250}
```

Comment:
out of curiosity - why pageSize 250 ?

Reply:
Mistakenly committed (used for experimenting with long lists of suggestions); will fix.